README.md

```diff
@@ -56,6 +56,7 @@ We would love it if you could help contributing to this repository.
 * [CoCo Lin](https://github.com/linmasaki)
 * [Dave Timmins](https://github.com/davetimmins)
 * [Dmitry Popov](https://github.com/justdmitry)
+* [Drew Killion](https://github.com/drewkill32)
```

Comment:
Please also add a new row to the table of providers for PingOne.

There's a template you can copy in a comment at the bottom of the table.